src/license/spdx/mod.rs

```diff
@@ -159,7 +159,7 @@ pub use license;
 mod tests;
 
 /// A boxed `license::License`.
-type BoxedLicense = Box<dyn license::License + Sync>;
+type BoxedLicense = Box<dyn license::License + Sync + Send>;
```

Comment:
This is fine, but I'm wondering why it came up as needed?

Reply:
This is needed for compatibility with crates that use `lazy static` with the `spin_no_std` feature.

Reply:
Ok. Might be worth a comment on the type def explaining why.